packages/client/src/impl/QueryApiImpl.ts

```diff
@@ -37,7 +37,6 @@ export default class QueryApiImpl implements QueryApi {
 
     this._transport = impl.queryTransport({
       host: host,
-      timeout: queryTimeout,
```

Comment:
I don't think we want to delete this.  It eventually gets passed to the `GrpcTransport` constructor as a field in `RpcOptions`.  Instead I think it should resolve to either the new `QueryOptions.timeout` value or to the ConnectionOptions `queryTimeout` field.